agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -500,13 +500,37 @@ void backhaul_manager::after_select(bool timeout)
         }
     }
 
+    // Send ap metrics query every reporting_interval_s seconds
+    static std::chrono::steady_clock::time_point ap_metric_interval =
+        std::chrono::steady_clock::now();
+    auto now     = std::chrono::steady_clock::now();
+    auto elapsed = std::chrono::duration_cast<std::chrono::seconds>(now - ap_metric_interval);
+    if (ap_metrics_reporting_info.reporting_interval_s != 0 &&
+        elapsed.count() >= ap_metrics_reporting_info.reporting_interval_s) {
+
+        ap_metric_interval = now;
+
+        std::vector<sMacAddr> bssid_list;
+        for (auto &socket : slaves_sockets) {
+            if (socket) {
+                for (int i = 0; i < beerocks::IFACE_TOTAL_VAPS; ++i) {
+                    if (socket->vaps_list.vaps[i].mac != network_utils::ZERO_MAC) {
+                        bssid_list.push_back(socket->vaps_list.vaps[i].mac);
+                    }
+                }
+            }
+        }
+
+        send_slave_ap_metrcic_query_message(0, bssid_list);
```

Comment:
There is a typo in the name of the method

Reply:
fixed